packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -49,8 +49,8 @@ export function Previewer({
 
   const remainingFilesLength = files.length - uploadedFilesLength();
   return (
-    <Card variation="outlined" className="amplify-fileuploader__previewer">
-      <Flex className="amplify-fileuploader__previewer__body">
+    <View className={ComponentClassNames.FileUploaderPreviewer}>
```

Comment:
Decided not to use the `Card` component in case we want to change the styles of the card and not affect this component. Really, this component is not just a composition of primitives like the Authenticator, but is it's own primitive in a way.